app/controllers/admin/processes_controller.rb

```diff
@@ -7,6 +7,8 @@ def index
     @processes = InstrumentProcess.all
   end
 
+  def show
+  end
```

Comment:
New line after end of method
```suggestion
  end
  
```

Reply:
Done